File: Kms.Security.Portal/app/nodak/common/company/company.dto.ts
module nodak.common.models {
    export class CompanyDto {
        private id: string;
        set Id(value) {
            this.id = value;
        }
        get Id() {
            return this.id;
        }

        private name: string;
        get Name() {
            return this.name;
        }
        set Name(value) {
            this.name = value;
        }

    }
}

File: Kms.Security.Common/DataContract/CompanyDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.Text;
using System.Threading.Tasks;

namespace Kms.Security.Common.Domain

{
   public class CompanyDto
    {
        [DataMember]
        public Guid Id { get; set; }

        [DataMember]
        public string Name { get; set; }
    }
}
